daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -196,7 +202,7 @@ object Node {
       with NodeInfo.Create {
 
     override def templateId: TypeConName = coinst.template
-
+    override def byKey: Boolean = false
```

Comment:
I find this confusing. Why would somebody ask a create note if it was "by key"?

Reply:
There is an example in the PR, in `EngineTest.scala`
```
 private[this] def byKeyNodes[Nid, Cid](tx: VersionedTransaction[Nid, Cid]) =
    tx.nodes.collect { case (nodeId, node) if node.byKey => nodeId }.toSet
```

Reply:
Oh, so it is mostly because of this old structure where the metadata contained information on whether a node was "by key" or not. That's not pretty but acceptable.